### PR TITLE
[RELEASE-ONLY] Update torch versions for 0.3 release

### PR DESCRIPTION
**Pull request by lucylq**

### CODE REVIEW

**Review by kirklandsign — APPROVED** [review]

File: install_requirements.sh [hunk 1]
```diff
@@ -85,7 +85,7 @@ DEVEL_REQUIREMENTS=(
 # TODO(dbort): Make each example publish its own requirements.txt
 EXAMPLES_REQUIREMENTS=(
   timm==0.6.13
-  torchaudio=="2.2.0.${NIGHTLY_VERSION}"
```
Comment (kirklandsign) [inline install_requirements.sh:88]: This should be in main as well?

**Review by kirklandsign — APPROVED** [review]

File: install_requirements.sh (hunk 1 above)
Comment (lucylq) [reply install_requirements.sh:88]: thanks - updated https://github.com/pytorch/executorch/pull/4000

Comment (lucylq) [reply install_requirements.sh:88]: note: merge when torch 2.4 pip package is released

Comment (kirklandsign) [reply install_requirements.sh:88]: OK seems that we need to merge this on release day

**Review by mergennachin — APPROVED** [review]
Thanks for the change

**Comment by mergennachin** [comment]
Can you test by running `./install_requirements.sh`?

**Comment by lucylq** [comment]
> Can you test by running `./install_requirements.sh` manually?

Yep - it builds well!

**Pull request merged**